sim/paladin/paladin.go

```diff
@@ -48,8 +48,8 @@ type Paladin struct {
 	Judgement         *core.Spell
 	DivineFavor       *core.Spell
 	// HolyShield            *core.Spell
-	// HammerOfWrath         *core.Spell
-	// HolyWrath             *core.Spell
+	// HammerOfWrath         []*core.Spell
+	// HolyWrath             []*core.Spell
```

Comment:
Should these be un-commented and assigned in their respective spell initializers?

Reply:
I think this pattern is outdated: these are assigned, but never used, since rotation isn't handled in code anymore.

There's even a pre-existing comment to this effect: "Active abilities and shared cooldowns that need externally manipulated". But almost none of these are "externally manipulated", luckily - since cross file references aren't making the code any easier to read.

I think these should be removed for all classes, unless such references are actually needed. And they should _definitely_ not be public.

Reply:
I guess yeah if they're not actively being used they aren't necessarily needed. They can always be added as needed. They just store pointers so they're not a massive memory hog but it's something